docs/cli/category-exporter.md

```diff
@@ -61,10 +61,10 @@ const options = {
     apiConfig: {
       apiUrl: 'https://api.europe-west1.gcp.commercetools.com'
       host: 'https://auth.europe-west1.gcp.commercetools.com'
-      project_key: 'PROJECT_KEY',
+      project_key: '<ProjectKey>',
```

Comment:
Suggestion is based on the code in the support ticket.

```suggestion
      projectKey: '<ProjectKey>',
```

Reply:
If `projectKey` is correct, then should the code examples in other Exporter documents also be changed?